services/src/main/resources/db/changelog/gamification.db.changelog-1.0.0.xml

```diff
@@ -296,17 +297,26 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
     <changeSet author="exo-gamification" id="1.0.0-17">
         <modifyDataType tableName="GAMIFICATION_ACTIONS_HISTORY"
                         columnName="USER_SOCIAL_ID"
-                        newDataType="NVARCHAR(200)"
-        />
+                        newDataType="NVARCHAR(200)"/>
     </changeSet>
 
     <changeSet author="exo-gamification" id="1.0.0-20">
-      <renameColumn oldColumnName="USER_SOCIAL_ID" newColumnName="EARNER_ID" columnDataType="BIGINT" tableName="GAMIFICATION_ACTIONS_HISTORY" />
-      <addColumn tableName="GAMIFICATION_ACTIONS_HISTORY">
-        <column name="EARNER_TYPE" type="INT" defaultValueNumeric="0">
-          <constraints nullable="false" />
-        </column>
-      </addColumn>
+        <renameColumn oldColumnName="USER_SOCIAL_ID" newColumnName="EARNER_ID" columnDataType="BIGINT"
+                      tableName="GAMIFICATION_ACTIONS_HISTORY"/>
+        <addColumn tableName="GAMIFICATION_ACTIONS_HISTORY">
+            <column name="EARNER_TYPE" type="INT" defaultValueNumeric="0">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+    </changeSet>
+
+    <changeSet author="exo-gamification" id="1.0.0-22">
```

Comment:
shouldn't the changeset ID be 1.0.0-21 ?

Reply:
work in progress still need this https://github.com/Meeds-io/gamification/pull/197 to be merged to continue working on this one

Reply:
done!